log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMMessageProcessing/js/VizGrammar.js

```diff
@@ -0,0 +1,2300 @@
+/*
```

Comment:
Hi Thanrik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/126

Thanks,
SajithD